### DISCUSSION TITLE
Help with auto imports and relative paths

### DISCUSSION THREAD

**Discussion by mitraecp** [nuxt/nuxt#17346] [Questions]
Hello guys, I'm having problems using auto imports, can anyone help me?

![image](https://user-images.githubusercontent.com/75841640/177217824-baa38a0f-d51f-4ef8-8b59-b44ed1857e90.png)

![image](https://user-images.githubusercontent.com/75841640/177217844-b09fc025-dc4c-4517-b262-a72c5949ea93.png)

**Comment by wobsoriano** [comment]
Run the app for it to generate the types. If the issue still persists then restart vscode

**Comment by Flanker72** [comment]
1. Check that you have [Volar extension](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar) installed.
2. Check that you have either [Take over mode](https://github.com/johnsoncodehk/volar/discussions/471) enabled or [TypeScript Vue plugin](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.vscode-typescript-vue-plugin) installed.
3. Run either `npm run dev` to start the dev server or `npx nuxi prepare` to generate typings.
4. Restart your IDE or press `Ctrl+Shift+P` to activate command palette and execute `Volar: Restart Vue server`, `ESLint: Restart ESLint server`, `TypeScript: Restart TS Server` if these commands exists.

This is because all typings and tsconfig are generated after running the commands from the step 3. If you have dev server started, all newly created/updated types will be automatically handled.
